.envrc

```diff
@@ -0,0 +1,2 @@
+use nix shell-eio.nix 
+eval $(opam env --shell=bash --inplace-path --set-switch)
```

Comment:
this shouldn't be committed. not every is using direnv.

Reply:
@bikallem this file still needs to be removed